Dockerfile

```diff
@@ -1,11 +1,26 @@
-FROM ghcr.io/comocheng/kineticmodelssite/kms-build:latest
+# Fetch RMG-Py build
+FROM comocheng/kms-rmgpy:latest as rmgpy
```

Comment:
`comocheng/kms-rmgpy` doesn't exist yet - this should be where the `Dockerfile.rmg` image should be built to